Removing flate2 dev dependency to avoid flate2 needing tokio feature.
--- a/Cargo.toml
+++ b/Cargo-patch.toml
@@ -56,10 +56,6 @@ version = "0.2.1"
 version = "0.4"
 default-features = false
 
-[dev-dependencies.flate2]
-version = "1"
-features = ["tokio"]
-
 [dev-dependencies.futures-cpupool]
 version = "0.1"
 
